### ISSUE TITLE
🚀 A listbox should accept a validationState

### ISSUE THREAD

**Issue by patricklafrance** [gsoft-inc/sg-orbit#618]
## Description

A listbox component selection could be invalid. A `validationState` prop is missing to indicate visually that a state is invalid.

The `aria-invalid` attribute should be supported.

## Checklist

- [x] Chromatic tests
- [x] Jest tests
- [x] Documentation
- [ ] Figma
